module_utils/ansible_nailgun_cement.py

```diff
@@ -56,6 +57,55 @@ class CommonParameter(
     pass
 
 
+class VMWareComputeResource(AbstractComputeResource):  # pylint:disable=R0901
```

Comment:
those two (this and `OVirtComputeResource`) are just temporarily here, and really belong into nailgun, right?